src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2936,8 +2936,12 @@ because xbuild doesn't support framework reference assemblies.
 		ToolPath="$(ZipAlignToolPath)"
 		ToolExe="$(ZipalignToolExe)"
 	/>
+	<ItemGroup>
+		<ApkAbiFilesSigned Include="$(ApkFileSigned)" Condition="'$(AndroidUseApkSigner)' == 'true'" />
+		<ApkAbiFilesSigned Condition="'$(AndroidCreatePackagePerAbi)' == 'true' And '$(AndroidUseApkSigner)' == 'true' " Include="$(OutDir)$(_AndroidPackage)*-Signed.apk" />
+	</ItemGroup>
 	<AndroidApkSigner Condition=" '$(AndroidUseApkSigner)' == 'true' "
-		ApkSignerJar="$(ApkSignerJar)"
+		ApkSignerJar="%(ApkAbiFilesSigned)"
```

Comment:
`ApkSignerJar` is the jar file for `apksigner`, if we passed an APK file here, it's probably going to fail...

Reply:
That's very true. I've just checked the version I'm using on my machine and I think I switched up the lines by acccident.